AxeObserver.js

```diff
@@ -59,36 +52,30 @@ export default class AxeObserver {
     // Configure axe
     axeCore.configure(axeCoreConfiguration)
   }
-  observe(targetNode, { debounceMs = 1000, maxWaitMs = debounceMs * 5 } = {}) {
+  observe(targetNode) {
     if (!targetNode) {
       throw new Error('AxeObserver.observe requires a targetNode')
     }
 
-    const scheduleAudit = debounce(
-      () => requestIdleCallback(() => this._auditTargetNode(targetNode)),
```

Comment:
Once the debounced function fired, we also scheduled the whole node audit via `requestIdleCallback`. While it was started during an idle period, the function would usually several hundred ms to execute. Hence the `requestIdleCallback` wasn't that useful, since the work would continue way past the actual idle period.

In the new version, smaller chunks of work are used. They get executed for only as long as the idle period lasts.